### ISSUE TITLE
Schemas should have a wiki page view

### ISSUE THREAD

**Issue by isoboroff** [cosmocode/dokuwiki-plugin-struct#318]
Right now, to view a schema I need to go into Admin and pull it up in the editor.  I should be able to see a schema as a wiki page.  This wiki page could be generated from the schema, not maintained manually as a parallel page.  If someone has a clue how to add this, I'd be happy to take a crack at it.

**Comment by annda** [comment]
I agree that the editor could be improved to make schema editing more transparent. Not yet sure if a wiki page view is the way to go.

**Comment by splitbrain** [comment]
I don't think this is about improving the schema editor. This is about making information about a scheme available to non-admin users.

I agree that often when creating a new aggregation, I have to go back to the schema editor to see what the columns are named and what type they have. Users don't have access to that.

So one solution could be a syntax component that simply lists all schema definitions. 

Alternatively it might also be enough if I could see the field names in addition to the labels when editing a page with an assigned schema?
